Userland/Applications/PixelPaint/main.cpp

```diff
@@ -541,6 +541,19 @@ int main(int argc, char** argv)
         },
         window));
 
+    layer_menu.add_action(GUI::Action::create(
+        "M&erge Active Layer Down", { Mod_Ctrl, Key_E }, [&](auto&) {
```

Comment:
Looks like Ctrl+E is copied from Photoshop? It doesn't make intuitive sense as a shortcut to me, but copying Photoshop is fair enough. :^)

Reply:
Yep, I'm not sure why it's Ctrl+E, but it's ingrained in my muscle memory so unless someone has a better idea I'm sticking to it